Update motivation_and_mindset.md (#26015)
TheOdinProject/curriculum: foundations/introduction/motivation_and_mindset.md

*   [Managing inspiration and motivation](https://markmanson.net/do-something)
*   [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)

> *   [Improve your typing skills with Keybr.com](https://www.keybr.com/). One excellent alternative is [typing.com](https://www.typing.com/). Another great option is [TypingClub.com](https://www.typingclub.com). Use these keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.

Reviewer: What do you think of this? It gets the main point of the bullet across and keeps all the suggestions in their own sub-bullets.
```suggestion
*   If you find your typing speed is holding you back, you can improve your typing skills by utilizing these keyboard trainers for at least 5 minutes every day:
  * [Keybr.com](https://www.keybr.com/)
  * [typing.com](https://www.typing.com/)
  * [TypingClub.com](https://www.typingclub.com)
```

Reviewer: Good call, I'll update my comment. I didn't notice.

*   [Managing inspiration and motivation](https://markmanson.net/do-something)
*   [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)

> *   If you find your typing speed is holding you back, you can improve your typing skills by utilizing these keyboard trainers for at least 5 minutes every day:
>   * [Keybr.com](https://www.keybr.com/)
>   * [typing.com](https://www.typing.com/)
>   * [TypingClub.com](https://www.typingclub.com)
>   * [Monkeytype](https://monkeytype.com/)

Reviewer 2: ```suggestion
-   If you find your typing speed is holding you back, you can improve your typing skills by utilizing these keyboard trainers for at least 5 minutes every day:
   - [Keybr.com](https://www.keybr.com/)
    - [typing.com](https://www.typing.com/)
    - [TypingClub.com](https://www.typingclub.com)
    - [Monkeytype](https://monkeytype.com/)
```
